configure.cmd

```diff
@@ -75,6 +75,21 @@ if not defined ARCH (
 
 set USE_CLANG_CL=0
 
+REM Some magic code to parse the parameters and extract those for CMake
+set CMAKE_PARAMS=
+set REMAINING=%*
+:loop
+for /f "tokens=1*" %%a in ("%REMAINING%") do (
```

Comment:
Wouldnt it be better to merge it into command line parsing thing below?

Reply:
Done.